pkg/controller/wip/controller_test.go

```diff
@@ -714,7 +714,6 @@ func newTestController(t *testing.T) (
 	}
 
 	stopCh := make(chan struct{})
-	informerFactory.Start(stopCh)
```

Comment:
If we don't have this, then we don't need the `stopCh`. Uses need to be cleaned up.